### PR TITLE
[new release] plotkicadsch and kicadsch (0.5.2)

### PR DESCRIPTION
**Pull request by jnavila**
Utilities to print and compare version of Kicad schematics

- Project page: <a href="https://jnavila.github.io/plotkicadsch/">https://jnavila.github.io/plotkicadsch/</a>
- Documentation: <a href="https://jnavila.github.io/plotkicadsch/index">https://jnavila.github.io/plotkicadsch/index</a>

##### CHANGES:

- Manage Fields with delimited strings
 - Use environment variables for internal differ
 - add dependency to an implementation of digestif

### CODE REVIEW

File: packages/kicadsch/kicadsch.0.5.2/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+build: [
+  [ "dune" "subst" ] {pinned}
+  [ "dune" "build" "-p" name "-j" jobs ]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/kicadsch/kicadsch.0.5.2/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune shouldn't be tagged as a build dependency (see #14088)

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+   
+build: [
+  [ "dune" "subst" ] {pinned}
+    [ "dune" "build" "-p" name ]
+]
+   
+depends: [
+  "dune" {build}
+    "kicadsch" {= "0.5.2"}
```
Comment (kit-ty-kate) [inline packages/plotkicadsch/plotkicadsch.0.5.2/opam]:
```suggestion
    "kicadsch" {= version}
```

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "tyxml" {>= "4.0.0"}
```
Comment (kit-ty-kate) [inline packages/plotkicadsch/plotkicadsch.0.5.2/opam]: could you unindent the next lines please, I'm not sure why everything is indented one way but not the first one. Indentation usually is 2.

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ * plotgitsch is able to compare git revisions of schematics
+"""
+   
+license: "ISC"
+   
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+   
+build: [
+  [ "dune" "subst" ] {pinned}
+    [ "dune" "build" "-p" name ]
```
Comment (kit-ty-kate) [inline packages/plotkicadsch/plotkicadsch.0.5.2/opam]:
```suggestion
  [ "dune" "build" "-p" name "-j" jobs]
```
This allows opam user to set the number of jobs that installation should use

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+]
+   
+depends: [
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/plotkicadsch/plotkicadsch.0.5.2/opam]:
```suggestion
  "dune" {>= "1.0"}
```
Dune shouldn't be tagged as a build dependency (see #14266)

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+license: "ISC"
+   
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+   
+build: [
+  [ "dune" "subst" ] {pinned}
+    [ "dune" "build" "-p" name ]
+]
+   
+depends: [
```
Comment (kit-ty-kate) [inline packages/plotkicadsch/plotkicadsch.0.5.2/opam]:
```suggestion
depends: [
  "ocaml"
```

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam (hunk 2 above)
Comment (jnavila) [reply packages/plotkicadsch/plotkicadsch.0.5.2/opam]: Ooooh. That's awesome!

File: packages/plotkicadsch/plotkicadsch.0.5.2/opam (hunk 6 above)
Comment (jnavila) [reply packages/plotkicadsch/plotkicadsch.0.5.2/opam]: I'm using dune-release. Shouldn't this be checked by the linter?

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**